### DISCUSSION TITLE
Support for Apple Safari

### DISCUSSION THREAD

**Discussion by stano-vittorio** [aclap-dev/video-downloadhelper#1368] [Help and Q&A] [answered]
### Your question

Hello everyone! I always use this app on chrome and firefox. I was wondering if there is a version (even beta) for safari and possibly if you plan to release it in the future. Thank you very much!

### Addon & CoApp version and details

N/D

**Comment by jcfrog** [comment, marked as answer]
Hi, sorry but nothing is planned for Safari at the moment.

**Reply by stano-vittorio** [reply]
Ok thanks for the feedback
